vizro-core/docs/pages/user_guides/filters.md

```diff
@@ -65,7 +65,8 @@ automatically based on the target column, e.g. a dropdown for categorical data o
 ## Changing Selectors
```

Comment:
```suggestion
## Changing selectors
```

Reply:
I'll be suggesting that we use sentence case (only first word capitalised) for subheadings and headings, but there's no style guide at present and there seems to be a mix on the site. Let's be consistent in this page and use it anyway, but no sweat about changing anything else until I run a consistency check across the docs 😅